packages/api/internal/edge/cluster.go

```diff
@@ -133,8 +134,12 @@ func (c *Cluster) GetAvailableTemplateBuilder(ctx context.Context) (*ClusterInst
 	span.SetAttributes(telemetry.WithClusterID(c.ID))
 	defer span.End()
 
-	var instances []*ClusterInstance
-	for _, instance := range c.instances.Items() {
+
+	// convert map to slice
+	mapItems := c.instances.Items()
+	instances:= make([]*ClusterInstance, 0, len(mapItems))
```

Comment:
Please run `make lint` and `make fmt` to make sure the styling and all lint rules are followed.
```suggestion
	instances := make([]*ClusterInstance, 0, len(mapItems))
```